test/models/setting_test.rb

```diff
@@ -3,24 +3,34 @@
 module Katello
   class SettingTest < ActiveSupport::TestCase
     def test_default_download_policy_setting
-      setting = Setting.where(:name => "default_download_policy").first
+      setting = Setting.where(:name => 'default_download_policy').first
       setting.value = "invalid"
       refute setting.valid?
       setting.value = "immediate"
       assert setting.valid?
     end
 
     def test_foreman_proxy_content_auto_sync_setting
-      setting = Setting.where(:name => "foreman_proxy_content_auto_sync").first
+      setting = Setting.where(:name => 'foreman_proxy_content_auto_sync').first
       setting.value = "invalid"
       refute setting.valid?
       setting.value = true
       assert setting.valid?
     end
 
+    def test_cdn_ssl_setting
+      setting = Setting.where(:name => 'cdn_ssl_version').first
+      setting.value = nil
+      assert setting.valid?
+      setting.value = 'SSLv23'
+      assert setting.valid?
+      setting.value = 'TLSv1'
```

Comment:
The tests here might be a little redundant. What I mean is that if we give it something bogus, like 'foo' will it be invalid? Checking both  SSLv23 and TLSv1 is not really increasing the coverage in any way as far as I can tell, but let me know if I'm missing something.

We can also increase the readability of the code by adding some whitespace:

```
   def test_cdn_ssl_setting
      setting = Setting.where(:name => 'cdn_ssl_version').first

      setting.value = nil
      assert setting.valid?

      setting.value = 'SSLv23'
      assert setting.valid?

      setting.value = 'TLSv1'
      assert setting.valid?
end
```